### PR TITLE
Added doc of chunk* options for addLayers

### PR DESCRIPTION
**Pull request by ghybs**
Description for addLayers options (`chunkedLoading`, `chunkInterval`, `chunkDelay` and `chunkProgress`).

Copied option `spiderLegPolylineOptions` into All options section as well (was only in top Defaults section).

Also slightly improved punctuation here and there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,11 +33,16 @@ By default the Clusterer enables some nice defaults for you:
 * **zoomToBoundsOnClick**: When you click a cluster we zoom to its bounds.
 * **spiderfyOnMaxZoom**: When you click a cluster at the bottom zoom level we spiderfy it so you can see all of its markers. (*Note: the spiderfy occurs at the current zoom level if all items within the cluster are physically located at the same latitude and longitude.*)
 * **removeOutsideVisibleBounds**: Clusters and markers too far from the viewport are removed from the map for performance.
+* **chunkedLoading**: When using addLayers, split the processing in small intervals so that the page does not freeze.
```
Comment (danzel) [inline README.md]: This isn't enabled by default

File: README.md [hunk 2]
```diff
@@ -61,15 +66,23 @@ Enabled by default (boolean options):
 * **zoomToBoundsOnClick**: When you click a cluster we zoom to its bounds.
 * **spiderfyOnMaxZoom**: When you click a cluster at the bottom zoom level we spiderfy it so you can see all of its markers. (*Note: the spiderfy occurs at the current zoom level if all items within the cluster are physically located at the same latitude and longitude.*)
 * **removeOutsideVisibleBounds**: Clusters and markers too far from the viewport are removed from the map for performance.
+* **chunkedLoading**: When using addLayers, split the processing in small intervals so that the page does not freeze. See also options chunkInterval, chunkDelay and chunkProgress below.
```
Comment (danzel) [inline README.md]: "in **to** small intervals"

Comment (danzel) [inline README.md]: Should probably put this next to the other chunk options

Comment (ghybs) [inline README.md]: Hi,

Thanks for the corrections, my bad!

I took also the opportunity to re-organize the Options and Methods sections.
For the latter, I tried to differentiate methods for the MCG from methods
for Clusters.

**Comment by ghybs** [comment]
Thanks for the corrections!

I also took the opportunity to re-organize the Options and Methods section.
For the latter, I tried differentiating methods for the MCG and the ones for Clusters only.

**Comment by danzel** [comment]
Awesome, thanks

**Pull request merged**